### PR TITLE
Add support for Issues

### PR DESCRIPTION
**Pull request by adambraimbridge**
Solves #28

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,77 +59,73 @@ ika --version
...
 ```bash
-ika create-project <github-url> [--body]
+ika projects create <github-url> [--body]
 
 # Create a new project.
 ```
 
-@see: [`src/commands/projects/README.md`](https://github.com/Financial-Times/ika/blob/master/src/commands/projects/README.md) for more details.
-
 ### Working with GitHub Pull Requests
```
Comment (taktran) [inline README.md]: Could you add documentation for `ika issues` too?

File: src/commands/projects.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * This project follows the example provided in the Yards documentation for command hierarchy and directory structure.
```
Comment (taktran) [inline src/commands/projects.js]:
```suggestion
 * This project follows the example provided in the Yargs documentation for command hierarchy and directory structure.
```

File: src/commands/pulls.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/**
+ * This project follows the example provided in the Yards documentation for command hierarchy and directory structure.
```
Comment (taktran) [inline src/commands/pulls.js]:
```suggestion
 * This project follows the example provided in the Yargs documentation for command hierarchy and directory structure.
```

File: README.md (hunk 1 above)
Comment (adambraimbridge) [reply README.md]: Most certainly.

**Pull request merged**